### PR TITLE
apply the remote config on store

### PR DESCRIPTION
**Pull request by Summys**

### CODE REVIEW

File: src/frontend/src/player.ts [hunk 1]
```diff
@@ -67,17 +67,18 @@ export class Player {
...
+  initStore(playerConfig: IPlayerStore): void {
     const shouldPersist = !storeManager.exists('player')
-    const store = storeManager.createStore<IPlayerStore>('player', DEFAULT_PLAYER_STORE)
 
-    if (shouldPersist) {
-      store.persist()
-    } else if (store.get('version') !== app.getVersion()) {
-      store.reset(DEFAULT_PLAYER_STORE)
+    if (storeManager.exists('player')) {
+      this.store.bulkSet(playerConfig)
```
Comment (lmancel) [inline src/frontend/src/player.ts]: There is no check to the values applied (both here and in flowrStore), which means that no matter the defaults they will be overridden by `null` values at first start if the configuration has not been done entirely (which will happen 90% of the time). We need to have a fallback mechanism to the default values if the ones provided by flowr-frontend are not defined.

**Review by lmancel — CHANGES_REQUESTED** [review]

File: src/frontend/src/player.ts (hunk 1 above)
Comment (Summys) [reply src/frontend/src/player.ts]: fixed by Dani

File: src/launcher/index.ts [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 import { clearBrowsingData } from '~/main/clearBrowsingData'
 import { getMigrateUserPreferences } from './migration/fromFlowrClientToFlowrPcClient'
 import { FlowrWindow } from 'src/frontend/flowr-window'
-export const log = require('electron-log')
 import { StoreManager, Store } from '../frontend/src/store'
 import { ApplicationManager } from '../application-manager/application-manager'
 import { IFlowrStore } from '../frontend/src/interfaces/flowrStore'
 import { keyboard } from '../keyboard/keyboardController'
+import { mergeWith, cloneDeep } from 'lodash'
+export const log = require('electron-log')
```
Comment (lmancel) [inline src/launcher/index.ts:14]: Does not seem to be used

File: src/frontend/src/player.ts [hunk 3]
```diff
@@ -67,17 +67,19 @@ export class Player {
...
-      store.reset(DEFAULT_PLAYER_STORE)
+    if (storeManager.exists('player')) {
+      const playerConfigMerged = mergeWith({}, this.flowrStore.data.player, DEFAULT_PLAYER_STORE, playerConfig, (a, b) => b === null || b === '' ? a : undefined)
+      this.store.bulkSet(playerConfigMerged)
     }
 
-    return store
+    if (shouldPersist) {
+      this.store.persist()
+    } else if (this.store.get('version') !== app.getVersion()) {
```
Comment (lmancel) [inline src/frontend/src/player.ts]: This `else if` statement is not necessary anymore. It is now obsolete with the sync from flowr-admin

**Review by lmancel — CHANGES_REQUESTED** [review]

File: src/launcher/index.ts (hunk 2 above)
Comment (Summys) [reply src/launcher/index.ts:14]: Are used .on `flowr-desktop-config`

**Pull request merged**